FV2310/IFTSTA/conditions.json

```diff
@@ -459,6 +459,16 @@
     "condition_text": "Wenn mehr als ein Grund vorliegt und die voranstehenden DE9013 dieses STS nicht ausreichen",
     "edifact_format": "IFTSTA"
   },
+  {
+    "condition_key": "144",
```

Comment:
das kann ich nicht verifizieren. in keinem der IFTSTA-Dokumente zum 01.Oktober 2023 steht eine bedingung 144 oder 145
![grafik](https://github.com/Hochfrequenz/edi_energy_ahb_conditions_and_packages/assets/23094997/5619553a-789a-4cdd-a8ed-e0ae5574ebe4)